core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournalLogWriter.java

```diff
@@ -263,9 +277,13 @@ private void maybeRotateLog() throws IOException {
   }
 
   private void createNewLogFile() throws IOException {
+    createNewLogFile(mNextSequenceNumber);
+  }
+
+  private void createNewLogFile(long startSequenceNumber) throws IOException {
     URI newLog = UfsJournalFile
-        .encodeLogFileLocation(mJournal, mNextSequenceNumber, UfsJournal.UNKNOWN_SEQUENCE_NUMBER);
-    UfsJournalFile currentLog = UfsJournalFile.createLogFile(newLog, mNextSequenceNumber,
+        .encodeLogFileLocation(mJournal, startSequenceNumber, UfsJournal.UNKNOWN_SEQUENCE_NUMBER);
+    UfsJournalFile currentLog = UfsJournalFile.createLogFile(newLog, startSequenceNumber,
```

Comment:
Lines 280-283 and 285-286 are added in this patch because of 1.8 specific issues.